### PR TITLE
Fixes/activity center UI issues

### LINKED ISSUE #14231: Activity Center is not closing upon tapping close button
## Problem 

Activity Center is utilising the `popover` view/component to display. The `:hide-popover` event is dispatched upon the user tapping the close button. However, the Activity Center is NOT closing.

### Reproduction

1) Open Status
2) Navigate to Activity Center
3) Try to close Activity Center

### Additional Information

- Operating System: Android, iOS

### PR DESCRIPTION
**Pull request by smohamedjavid**
Fixes #14231, #14241 and #14243

### Summary

Contains fixes for Activity Center UI issues. 

**Design** 

([Figma Link](https://www.figma.com/file/eDfxTa9IoaCMUy5cLTp0ys/Shell-for-Mobile?node-id=3806%3A586138))

#### Areas impacted

- Activity Center (Notifications Center)

**Platforms**

- Android
- iOS

**Steps to test**

1. Open the `Status` application
2. Navigate to the `Communities` or `Messages` tab
3. Open  `Activity Center`

status: ready

### CODE REVIEW

**Review by ilmotta — APPROVED** [review]
Code-wise it LGTM, but I haven't actually tested the UI changes.

File: src/status_im/ui/screens/activity_center/views.cljs [hunk 1]
```diff
@@ -219,19 +219,20 @@
...
-          [header]
-          [rn/flat-list {:content-container-style {:flex-grow 1}
-                         :data                    notifications
-                         :empty-component         [empty-tab]
-                         :key-fn                  :id
-                         :on-end-reached          #(>evt [:activity-center.notifications/fetch-next-page])
-                         :render-fn               render-notification}]]]))}))
+  [:f>
+   (fn []
+     (let [notifications         (<sub [:activity-center/filtered-notifications])
```
Comment (ilmotta) [inline src/status_im/ui/screens/activity_center/views.cljs:224]: [Code style] Let bindings have extra space.

Comment (smohamedjavid) [reply src/status_im/ui/screens/activity_center/views.cljs:224]: removed it

**Comment by pavloburykh** [comment]
@smohamedjavid thank you for the PR. 

As far as I understand current PR introduces fixes of the NEW UI Activity center. 
At the same time PR builds still have OLD UI Activity center as well as nightly builds. 

Could you please elaborate on this? Maybe I have missed some PR that has introduced NEW UI Activity center? Or it should be introduced by current PR together with the fixes? Thanx!

photo_2022-10-27 13 32 50

https://user-images.githubusercontent.com/97245802/198262172-9fb98d45-de91-4398-9cf9-d8e2831d9c6e.mp4

**Comment by smohamedjavid** [comment]
@pavloburykh - The new Activity Center is not enabled yet for testing as there is development underway (and bug fixes too) and will be enabled in 2-3 weeks time.

**Comment by pavloburykh** [comment]
> @pavloburykh - The new Activity Center is not enabled yet for testing as there is development underway (and bug fixes too) and will be enabled in 2-3 weeks time.

got it. thank you @smohamedjavid

**Pull request merged**